### ISSUE TITLE
Fix missing Legacy cases after case is signed

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#13872]
## Description
After a judge signs a case, when they return to their queue view, legacy cases are no longer listed until the page is reloaded/refreshed.

## Acceptance criteria
- [ ] Legacy cases are present (without a manual page refresh) in the judges queue view after the judge signs a case

## Background/context/resources
Per VLJ User: 
"When I finish signing a case, Caseflow loads a page that shows the cases I have available pending my signature.  Lately, that list only shows AMA appeals and not Legacy appeals.  If I refresh the list, then the list will include all (AMA and Legacy) appeals.  Other Judges have experienced the same issue so I do not think it is unique to me."

## Technical notes
Possibly related to #13664

**Comment by jjohni** [comment]
YourIT Ticket Number | Date Ticket Created | Case Details URL
-- | -- | --
INC0000000 | 00/00/00 | Add case details page URL here
INC10192233 | 04/03/2020 | On the case details page of there queue product
INC10176192 | 04/02/2020 | On the case details page of there queue product
INC10177256 | 04/02/2020 | On the case details page of there queue product
INC10169579 | 04/01/2020 | On the case details page of there queue product

**Comment by ajspotts** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | ||||
3 | ||||||||
5 | 
8 | 
```

Why 2?
- Wishful thinking? May be a small code change.

Why 3?
- State management can be complicated in our app

**Comment by hschallhorn** [comment]
😕  Having a hard time reproducing this again. Steps I followed:
1. Log in as bvaaabshire
1. Go to their queue
1. Click a legacy case
1. Select ready for dispatch
1. Continue through checkout
1. Ensure all legacy cases appear in queue

Also tested searching for a case so that tasks were not loaded through queue before dispatching a case. Looks like everything loads correctly.

After sending to dispatch, before refresh|After refresh
---|---
![Screen Shot 2020-09-29 at 11 16 04 AM](https://user-images.githubusercontent.com/45575454/94578125-52cb8280-0245-11eb-8cc3-637b4bcf4a55.png)|![Screen Shot 2020-09-29 at 11 16 10 AM](https://user-images.githubusercontent.com/45575454/94578131-552ddc80-0245-11eb-988e-158b56fbf723.png)

Thoughts:
This may have been resolved by #14252. One of the changes was that every time queue is loaded, we pull tasks from the back end.
